pages/query.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from "@tanstack/react-query";
+
+const getComments = async () => {
+  const COMMENTS_URL = "https://jsonplaceholder.typicode.com/comments";
+
+  const res = await fetch(`${COMMENTS_URL}`);
+  return res.json();
+};
+export const useComments = () => {
+  const { data, error, isLoading } = useQuery({
+    queryKey: ["comments"],
```

Comment:
I still want you to create a file where you export all your queryKeys from.

So maybe a `queryKeys.ts` file that would export an object that has `QUERY_KEY: 'query_key'` pairs, for example
```js
const queryKeys = {
COMMENT: 'comment'
}
```

This would serve as a source of truth for every query key you would be using, even you mutation keys.

Reply:
ok